index.html

```diff
@@ -11,26 +11,141 @@
+                <li class="languages">javascript</li>
+            </ul>
+            <p class="worksee" >See Project</p>
+        </article>
+
+    <article class="work">
+        <header class="work">
+            <img class="work" src="Images/Snapshoot-Portfolio3.png" alt="GymFit"/>
+            <h1 class="work">Tonic</h1>
+        </header>
+        <ul class="worklist">
+            <li class="card-title">CANOPY</li>
+            <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+            <li class="work">Back End Dev</li>
+            <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+            <li class="work">2015</li>
+        </ul>
+        <p class="work">A daily selection of privately personalized reads; no accounts 
+            or sign-ups required.</p>
+        <ul class="languages">
+            <li class="languages">html</li>
+            <li class="languages">css</li>
+            <li class="languages">javascript</li>
+        </ul>
+        <p class="worksee" >See Project</p>
+    </article>
+
+    <article class="work">
+        <header class="work">
+            <img class="work" src="Images/Snapshoot-Portfolio4.png" alt="Professional Art Printing"/>
+            <h1 class="work">Multi-Post Stories</h1>
+        </header>
+        <ul class="worklist">
+            <li class="card-title">CANOPY</li>
+            <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+            <li class="work">Back End Dev</li>
+            <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+            <li class="work">2015</li>
+        </ul>
+        <p class="work">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="languages">
+            <li class="languages">html</li>
+            <li class="languages">css</li>
+            <li class="languages">javascript</li>
+        </ul>
+        <p class="worksee" >See Project</p>
+    </article>
+
+    <article class="AboutMe">
+        <h1 class="AboutMe">About myself</h1>
+        <p class="AboutMe">
+         I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>     
+        <h3>Let's connect</h3> 
         <ul class="social">
             <li><img class="button" src="Images/Vector-3.svg" alt="Twitter Icon"/></li>
             <li><img class="button" src="Images/Linkedin-icon.svg" alt="Linkedin Icon"/></li>
             <li><img class="button" src="Images/Vector-1.svg" alt="Gmail Icon"/></li>
             <li><img class="button" src="Images/Vector-2.svg" alt="Cat Icon"/></li>
             <li><img class="button" src="Images/Vector.svg" alt="Peace Icon"/></li>
         </ul>
-    </section>
-    
+        <p class="worksee" >Get my resume</p>
```

Comment:
- [ ] Please try to wrap get my resume inside a `button` tag because it will make your project look more professional and dynamic as it will be responding to interactions